src/frontend/src/binder/update.rs

```diff
@@ -219,7 +219,8 @@ impl Binder {
             .iter()
             .enumerate()
             .filter_map(|(i, c)| {
-                (!c.is_generated()).then_some(InputRef::new(i, c.data_type().clone()).into())
+                c.can_dml()
```

Comment:
Does it mean that we (previously) also allow DML on `__row_id` column? Shall we ban this as well?

Reply:
We allow DML like deletes on `_row_id` column. I think no need to ban it.